src/views/AdminPage.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import DogForm from '../Components/DogForm';
+import { createDog } from '../services/dogs';
+import { useHistory } from 'react-router-dom/cjs/react-router-dom.min';
+
+export default function AdminPage() {
+  const [name, setName] = useState('');
+  const [age, setAge] = useState('');
+  const [bio, setBio] = useState('');
+  const [breed, setBreed] = useState('');
+  const [image, setImage] = useState('');
+  const [treat, setTreat] = useState('');
+  const [error, setError] = useState('');
+  const history = useHistory();
+
+  const handleSubmit = async () => {
+    try {
+      await createDog({ name, age, bio, breed, image, treat });
+      history.push('/dogs');
+    } catch (e) {
```

Comment:
Here we will want to add a success message after the user successfully adds a dog.